VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -64,24 +73,24 @@ final internal class ResponseCache
     {
         let key = request.cacheKey
         
-        self.memoryCache.removeResponseDictionaryForKey(key)
```

Comment:
I'm using the "responseDictionary" name since we're dealing with dictionaries rather than Responses. It might be weird if we're telling it to remove a "response" which is actually a dictionary and not a Response. Thoughts?